test/timecop_test.rb

```diff
@@ -689,8 +689,106 @@ def test_thread_safe_timecop_returns_after_block
     Timecop.thread_safe = false
   end
 
+  if RUBY_VERSION >= '2.1.0'
+    def test_process_clock_gettime_monotonic
+      Timecop.freeze do
+        assert_same monotonic, monotonic, "CLOCK_MONOTONIC is not frozen"
+      end
+
+      current = monotonic
+      Timecop.freeze(-0.5) do
+        assert monotonic < current, "CLOCK_MONOTONIC is not traveling back in time"
+      end
+    end
+
+    def test_process_clock_gettime_monotonic_nested
+      Timecop.freeze do
+        parent = monotonic
+
+        sleep(0.1)
+
+        Timecop.freeze(0.5) do
+          child = monotonic
+          assert_times_effectively_equal(child, parent + 0.5, 0.001, "Nested freeze not working for monotonic time")
+        end
+      end
+    end
+
+    def test_process_clock_gettime_monotonic_travel
+      current = monotonic
+      Timecop.travel do
+        refute_same monotonic, monotonic, "CLOCK_MONOTONIC is frozen"
+        sleep 0.5
+        assert monotonic > current, "CLOCK_MONOTONIC is not moving forward"
+      end
+
+      Timecop.travel(-0.5) do
+        refute_same monotonic, monotonic, "CLOCK_MONOTONIC is frozen"
+        sleep 0.5
+        assert monotonic > current, "CLOCK_MONOTONIC is not traveling properly"
+      end
+    end
+
+    def test_process_clock_gettime_monotonic_scale
+      scale = 4
+      sleep_length = 0.25
+      Timecop.scale(scale) do
+        current = monotonic
+        sleep(sleep_length)
+        expected_time = current + (scale * sleep_length)
+        assert_times_effectively_equal expected_time, monotonic, 0.1, "CLOCK_MONOTONIC is not scaling"
+      end
+    end
+
+    def test_process_clock_gettime_realtime
+      Timecop.freeze do
+        assert_same realtime, realtime, "CLOCK_REALTIME is not frozen"
+      end
+
+      current = realtime
+      Timecop.freeze(-20) do
+        assert realtime < current, "CLOCK_REALTIME is not traveling back in time"
+      end
+    end
+
+    def test_process_clock_gettime_realtime_travel
+      current = realtime
+      Timecop.travel do
+        refute_equal realtime, realtime, "CLOCK_REALTIME is frozen"
+        assert realtime > current, "CLOCK_REALTIME is not moving forward"
+      end
+
+      Timecop.travel(Time.now - 0.5) do
+        refute_equal realtime, realtime, "CLOCK_REALTIME is frozen"
+        assert realtime < current, "CLOCK_REALTIME is not traveling properly"
+        sleep 0.5
+        assert realtime > current, "CLOCK_REALTIME is not traveling properly"
+      end
+    end
+
+    def test_process_clock_gettime_realtime_scale
+      scale = 4
+      sleep_length = 0.25
+      Timecop.scale(scale) do
+        current = realtime
+        sleep(sleep_length)
+        assert current + scale * sleep_length < realtime, "CLOCK_REALTIME is not scaling"
+      end
+    end
+  end
+
   private
 
+  if RUBY_VERSION >= '2.1.0'
```

Comment:
I know this is just following the convention of this codebase, so this isn't a comment on your PR per se:

Ruby 2.1.0 is 11 years old and has been [EOL for 7 years](https://endoflife.date/ruby).

Reply:
It would be nice to update the minimum Ruby version in the gemspec; right now it's at 1.9.2.